### PR TITLE
Support `EXPLAIN` / `DESCR` / `DESCRIBE` `[FORMATTED | EXTENDED]`

### PR DESCRIPTION
**Pull request by jonathanlehto**
adding extended and formatted

Hive EXTENDED / FORMATTED -- https://docs.cloudera.com/cdw-runtime/cloud/materialized-view-commands/topics/hive_describe_materialized_view.html

### CODE REVIEW

File: src/ast/mod.rs [hunk 1]
```diff
@@ -2190,7 +2190,8 @@ pub enum Statement {
     /// Note: this is a MySQL-specific statement. See <https://dev.mysql.com/doc/refman/8.0/en/explain.html>
     ExplainTable {
         /// If true, query used the MySQL `DESCRIBE` alias for explain
-        describe_alias: bool,
+        describe_alias: DescribeAlias,
+        hive_format: Option<HiveDescribeFormat>,
```
Comment (alamb) [inline src/ast/mod.rs:2194]:
```suggestion
        /// Hive style `FORMATTED | EXTENDED` 
        hive_format: Option<HiveDescribeFormat>,
```

**Review by alamb — APPROVED** [review]
As always thanks again. I have a few small comment improvements, but I'll plan to make them myself

File: src/ast/mod.rs [hunk 2]
```diff
@@ -2190,7 +2190,9 @@ pub enum Statement {
     /// Note: this is a MySQL-specific statement. See <https://dev.mysql.com/doc/refman/8.0/en/explain.html>
     ExplainTable {
         /// If true, query used the MySQL `DESCRIBE` alias for explain
```
Comment (alamb) [inline src/ast/mod.rs:2192]:
```suggestion
        /// `EXPLAIN | DESC | DESCRIBE````

**Comment by jonathanlehto** [comment]
Thank you @alamb !  I appreciate the changes!

**Pull request merged**